src/components/PlaidLink/index.js

```diff
@@ -1,44 +1,78 @@
-import {useCallback, useEffect} from 'react';
-import {usePlaidLink} from 'react-plaid-link';
+import React from 'react';
+import {View} from 'react-native';
+import styles from '../../styles/styles';
 import {plaidLinkPropTypes, plaidLinkDefaultProps} from './plaidLinkPropTypes';
-import Log from '../../libs/Log';
-
-const PlaidLink = (props) => {
-    const onSuccess = useCallback((publicToken, metadata) => {
-        props.onSuccess({publicToken, metadata});
-    }, []);
-
-    const {open, ready, error} = usePlaidLink({
-        token: props.token,
-        onSuccess,
-        onExit: (exitError, metadata) => {
-            Log.info('[PlaidLink] Exit: ', false, {exitError, metadata});
-            props.onExit();
-        },
-        onEvent: (event, metadata) => {
-            Log.info('[PlaidLink] Event: ', false, {event, metadata});
-        },
-
-        // The redirect URI with an OAuth state ID. Needed to re-initialize the PlaidLink after directing the
-        // user to their respective bank platform
-        receivedRedirectUri: props.receivedRedirectURI,
```

Comment:
Are these changes affecting OAuth and did we test that things still work?

Reply:
was OOO, looking into this now

Reply:
I reverted the changes because, according to the plaid docs, its not recommended to use iframes, and also iframes can't store session info, so my best best was to use the `forceMobile: true`  in `usePlaidLink` hook, which wasn't possible, so i created a PR to `react-plaid-link` https://github.com/plaid/react-plaid-link/pull/260. If this is merged we can go ahead with this PR, until then am putting this on hold.

Reply:
Oh interesting. Did anyone from Plaid give an indication that they would accept a PR with those changes? They look nice, but I wonder if we should raise an issue first.

Reply:
Nope, but i saw some open PR's so i decided to add mine